examples/resources/fmc_hosts/import.sh

```diff
@@ -0,0 +1,2 @@
+# Modified manually, we want to provide domain and multiple hosts to import
+terraform import fmc_hosts.example "Domain|host1,host2,..."
```

Comment:
I would keep the comma to separate different identifiers and use array syntax to allow providing multiple ids of the same type:

```
terraform import fmc_hosts.example "<domain>,[<host>]"
```

so you could then import using something like this:

```
terraform import fmc_hosts.example "Domain1,[Host1,Host2]"
```

But this is something we need to implement at the template level, because otherwise, as you suggested, those manual changes would always be overwritten. Also the ability to specify the domain is missing for all the existing resources.

Reply:
Import template updated. I believe domain support for existing objects needs to be handled separately.